src/util/filepathurl.mm

```diff
@@ -0,0 +1,35 @@
+#include "util/filepathurl.h"
+
+
+
+#import <Foundation/Foundation.h>
+
+
+// Taken from src/platformsupport/clipboard/qmacmime.mm
```

Comment:
A reference of the Qt version from which this has been copied/backported might be helpful. The plain path even doesn't reveal that this is actually source code from Qt. Please use an URL.